### PR TITLE
Added Utils layer examples

### PR DESCRIPTION
**Pull request by aeharvlee**
## Proposed changes

This PR added examples related with [Utils Layer in Common Architecture](https://kips.klaytn.com/KIPs/kip-34#utils-layer).

When you review this PR, please check the comments placed at the top of the main function. There are related articles. 
I wrote that comments here also for your convenience.

### utils/play_with_units_in_klaytn
* Check [Boilerplate.java](https://github.com/aeharvlee/caver-java-boilerplate/blob/a2647304da07a6688710a3517597a98b60f17fb3/utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java) directly. You don't have to check Gradle components, which is automatically created.
* Related reference - Korean: [KLAY의 단위](https://ko.docs.klaytn.com/bapp/sdk/caver-js/getting-started#sending-klay-at-a-glan)
* Related reference - English: [Units of KLAY
](https://docs.klaytn.com/bapp/sdk/caver-js/getting-started#sending-klay-at-a-glance)

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [x] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-java-examples)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                        .setValue(BigInteger.ONE)
+        );
+        // Example-1: Sending 0.5 KLAY to recipient
+        // 1 KLAY is actually 10^18(=1000000000000000000) peb. So if you want send 0.5 KLAY,
+        // option-1: Set actual peb value directly to ValueTransfer transaction instance.
+        vt.setValue(BigInteger.valueOf(500000000000000000L)); // 5 * (10^17)
+        System.out.println("Example-1) The value what we set using option-1 is " + vt.getValue());
+
+        // option-2 (Recommended): option-1 is very cumbersome and difficult to read. Let's use more elegant way.
+        String KLAY = caver.utils.convertToPeb(BigDecimal.valueOf(5), Utils.KlayUnit.KLAY);
```
Comment (sirano11) [inline utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java]: Please use lower case.

Comment (aeharvlee) [reply utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java]: Thanks for your feedback. I will. 👍

Comment (aeharvlee) [reply utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java]: @sirano11 
I updated it here.
Please review this [file](https://github.com/klaytn/caver-java-examples/blob/4757e5b9399fd447a242815f1cc6eefc05a87335/utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java) :)

File: utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+        System.out.println("Example-1) The value what we set using option-1 is " + vt.getValue());
+
+        // 1 KLAY is actually 10^18(=1000000000000000000) peb. So if you want send 0.5 KLAY,
+        // option-2 (Not recommended): Set actual peb value directly to ValueTransfer transaction instance.
+        vt.setValue(BigInteger.valueOf(500000000000000000L)); // 5 * (10^17)
+        System.out.println("Example-1) The value what we set using option-2 is " + vt.getValue());
+
+        // Example-2: Sending 0.05 KLAY to recipient
+        // option-1 (Recommended): Use KlayUnit.
+        vt.setValue(new BigDecimal(caver.utils.convertToPeb("0.05", Utils.KlayUnit.KLAY)).toBigInteger());
```
Comment (sirano11) [inline utils/play_with_units_in_klaytn/src/main/java/Boilerplate.java:99]: Why did you use .toBigInteger?? is it a same code at Line 109?

**Review by sirano11 — APPROVED** [review]

**Pull request merged**